concent_api/verifier/tasks.py

```diff
@@ -114,11 +119,9 @@ def blender_verification_order(
             raise
 
     # Verifier unpacks the archive with project source.
-    for file_path in (source_package_path, result_package_path):
+    for file_path, download_file_name in package_paths_to_downloaded_file_names.items():
         try:
-            unpack_archive(
-                os.path.basename(file_path)
-            )
+            unpack_archive(download_file_name)
         except (OSError, BadZipFile) as e:
```

Comment:
What kind of `OSError` are you expecting here? I think you should crash on `OSError`.

Reply:
Answered in https://github.com/golemfactory/concent/pull/441